app/views/defence_requests/form_partials/_detainee_details.html.erb

```diff
@@ -24,8 +24,8 @@
       <%= t('transgender') %>
     </label>
     <label class="block-label form-label" for="defence_request_gender_not_specified">
```

Comment:
This label is now for a wrong field. It should be `..._gender_unspecified`. Clicking on the label doesn't work :) Maybe we should always use the `f.label` helper?